### PR TITLE
add scroll function on the click link

### PR DESCRIPTION
**Pull request by jason-jz-zhu**
we can scroll back to top.

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -2,46 +2,52 @@ import React, { PropTypes } from 'react';
...
-          <Link to={props.preLink}>
+const Footer = (props) => {
+  const scrollFunction = (event) => {
+    document.body.scrollTop = 0;
+    document.documentElement.scrollTop = 0;
+  };
+  return (
+    <div className="footer">
+      <div className="nav-footer">
+        { /* pre page button */  }
```
Comment (CDEMICHIEI) [inline src/components/Footer/Footer.js:13]: Include the angle marks as clickable links.

File: src/components/Footer/Footer.js [hunk 2]
```diff
@@ -2,46 +2,52 @@ import React, { PropTypes } from 'react';
...
+          {props.preTitle !== undefined ?
+            <i className="fa fa-angle-left fa-3x" aria-hidden="true"></i> : null}
+        </div>
+        { /* the verticality line */  }
+        <div className="vLine" />
+        { /* right angule mark */  }
+        <div className="right-angle">
+          <i className="fa fa-angle-right fa-3x" aria-hidden="true"></i>
+        </div>
+        { /* next page button */  }
```
Comment (CDEMICHIEI) [inline src/components/Footer/Footer.js:33]: Add similar logic to `next-page` div as you have in `prev-page` to handle case when `props.afterLink` is undefined.

File: src/components/Footer/Footer.js [hunk 3]
```diff
@@ -2,46 +2,52 @@ import React, { PropTypes } from 'react';
...
+const Footer = (props) => {
+  const scrollFunction = (event) => {
+    document.body.scrollTop = 0;
+    document.documentElement.scrollTop = 0;
+  };
+  return (
+    <div className="footer">
+      <div className="nav-footer">
+        { /* pre page button */  }
+        <div style={props.preTitle !== undefined && props.preContent.length > 8 ? { width: '100px' } : { width: '60px' } }>
```
Comment (CDEMICHIEI) [inline src/components/Footer/Footer.js:14]: Add className for `pre-page`
I don't understand the need for a check on `length` here.

File: src/components/Footer/Footer.js [hunk 4]
```diff
@@ -2,46 +2,52 @@ import React, { PropTypes } from 'react';
...
 
-const Footer = (props) => (
-  <div className="footer">
-    <div className="nav-footer">
-      { /* pre page button */  }
-      <div style={props.preTitle !== undefined && props.preContent.length > 8 ? { width: '100px' } : { width: '60px' } }>
-        {props.preTitle !== undefined ?
-          <Link to={props.preLink}>
+const Footer = (props) => {
+  const scrollFunction = (event) => {
```
Comment (CDEMICHIEI) [inline src/components/Footer/Footer.js]: Scrolling works great!

**Review by CDEMICHIEI — COMMENTED** [review]
Scrolling works as intended, and I found a few places to improve usability in the footer.

File: src/components/Footer/Footer.js (hunk 3 above)
Comment (jason-jz-zhu) [reply src/components/Footer/Footer.js:14]: We have different length of word. For example, "PowerPoint" is almost twice longer than "Citation".
If the longer word is in the pre-page, we need to make width larger than before.

You can check the email page and PowerPoint to see the difference.

File: src/components/Footer/Footer.js (hunk 2 above)
Comment (jason-jz-zhu) [reply src/components/Footer/Footer.js:33]: Cool. This is a good point. I will fix it.

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (jason-jz-zhu) [reply src/components/Footer/Footer.js:13]: We can show it at tmr meeting. And then we can add this feature at next sprint. Thanks.

**Review by CDEMICHIEI — APPROVED** [review]
Looks good

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (CDEMICHIEI) [reply src/components/Footer/Footer.js:13]: Sounds good

**Pull request merged**